projects/testing-library/src/lib/testing-library.ts

```diff
@@ -176,7 +191,7 @@ export async function render<SutType, WrapperType = SutType>(
     detectChanges,
     navigate,
     rerender,
-    debugElement: fixture.debugElement.query(By.directive(sut)),
+    debugElement: typeof sut === 'string' ? fixture.debugElement : fixture.debugElement.query(By.directive(sut)),
```

Comment:
This is an only difference between `render(directive, { template })` and `render(template)`. Is this acceptable?

Reply:
Yes, this fine.